src/main/java/app/service/impl/OperationBuy.java

```diff
@@ -0,0 +1,32 @@
+package app.service.impl;
+
+import app.FruitStorage;
+import app.model.SupplyFruitBatch;
+import app.service.Operation;
+import java.util.List;
+
+public class OperationBuy implements Operation {
+
+    @Override
+    public void execute(List<String> data) {
+        SupplyFruitBatch currentFruit = new FruitParserImplementation().parse(data);
+        for (SupplyFruitBatch iteratorFruit : FruitStorage.SUPPLY_FRUIT_BATCHES) {
+            if (iteratorFruit.getFruitName().equals(currentFruit.getFruitName())
```

Comment:
can we move it to a separate method? This `if` statement is too complex for reading